src/workers/postgres.rs

```diff
@@ -327,37 +388,60 @@ impl Postgres {
                     }
                 }
 
-                let Ok(session) = self.get_session().await else {
+                if tx_batch.is_empty() && block_batch.is_empty() && update_batch.is_empty() {
+                    tokio::time::sleep(Duration::from_millis(400)).await;
```

Comment:
lower sleep here to 50ms, no need to wait so long

Reply:
fixed